styling/Phoenix_files/_hero.scss

```diff
@@ -84,3 +85,25 @@ body, h1, h2, h3, h4, h5, h6 {
   background-image: url("../Phoenix_files/phoenix-logo-white.png") !important;
   background-size: 250px 34px;
 }
+
+#subheader-columns {
+  margin: 50px 30px 0; }
+
+.subheader h4 {
+  color: #000;
```

Comment:
This should probably introduce a new variable like `$body-color` so we can use it in multiple places. What do you think? I think it's in variables.css